src/Tools/dotnet-counters/ConsoleWriter.cs

```diff
@@ -72,16 +74,17 @@ public void ToggleStatus(bool pauseCmdSet)
 
         public void Update(string providerName, ICounterPayload payload, bool pauseCmdSet)
         {
-            if (paused != pauseCmdSet)
+
+            if (!initialized)
             {
-                ToggleStatus(pauseCmdSet);
+                initialized = true;
+                UpdateStatus("    Status: Running");
```

Comment:
What happens when the specified provider does not exist in the runtime? Would the status show running?

Reply:
No. It'll show "waiting for initial data"